admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js

```diff
@@ -286,6 +287,9 @@ function (ich,Marionette,Backbone,ConfigurationEdit,Service,Utils,wreqr,_,$,moda
             var properties = config.get('properties');
             return properties.get('shortname') || properties.get('id');
         },
+        getPID: function(config) {
```

Comment:
I certainly could inline this. When I stuck it in initially, I thought there'd be more to it.